documentation/development/docker.md

```diff
@@ -0,0 +1,59 @@
+# Docker
+
+This describes how to create an environment for development using Docker.
+
+**This is still experimental, using [Vagrant](../installation.md) is much better tested and supported!**
+
+## Running OCIM in a docker container
+
+First, set your `.env` file as you would normally:
```

Comment:
Do we not have a working example of this .env file (for dev, not the test or e2e example), private or public somewhere? I thought we did. Doesn't need to block this PR though.

Reply:
I found [this](https://github.com/open-craft/opencraft/blob/f327ff0c1fd9f406f459ea790f1222b161f7c434/documentation/configuration.md#configuration) in the documentation.

Reply:
Ah, thanks :)